src/client/main.cpp

```diff
@@ -150,7 +150,7 @@ int main(int argc, const char **argv)
 
 
 	irr::IrrlichtDevice *device = irr::createDevice(
-	  irr::video::EDT_OPENGL, irr::core::dimension2d<irr::u32>(1280, 720), 16, false, false, false);
+	  irr::video::EDT_OPENGL, irr::core::dimension2d<irr::u32>(1920, 1080), 16, true, false, false);
```

Comment:
This crashes my window manager (assuming it's fullscreen like I think it is), so could we change this to be some sort of command line flag to enable it, and then just add that to the lab machine init script?

Reply:
yeah this for fullscreen

Reply:
Yeah it's 1080 fullscreen.. I had a feeling this might break your quite frankly __ludicrous__ setup 😆

Reply:
Yeah, because I previously mentioned it when we last tested it